app/assets/javascripts/sprockets.js

```diff
@@ -16,12 +16,6 @@
 //= require select2
 //= require_tree .
 
-$(function () {
```

Comment:
Moved to a similar method to attach to `#new-public-inks` as other parts of the app, to avoid a method on `window`.